src/test/resources/json/sample.json

```diff
@@ -382,6 +382,9 @@
                                     "status": "failed"
                                 }
                             }
+                        ],
+                        "comments": [
+                            "Setting the account balance to 100"
```

Comment:
since this is an array I'd like to have example with single and with multiply comments and both should be examined by integration tests